agent/lib/kontena/workers/image_pull_worker.rb

```diff
@@ -4,6 +4,8 @@ class ImagePullWorker
     include Celluloid
     include Kontena::Logging
 
+    exclusive :ensure_image
```

Comment:
We should remove entry from the cache if pull fails, how would you handle it here?